devtools/gradle/application-plugin/build.gradle

```diff
@@ -0,0 +1,27 @@
+dependencies {
+    api gradleApi()
+
+    implementation "io.quarkus:quarkus-bootstrap-core:${version}"
+    implementation "io.quarkus:quarkus-devtools-common:${version}"
+    implementation "io.quarkus:quarkus-core-deployment:${version}"
+
+    testImplementation "io.quarkus:quarkus-project-core-extension-codestarts:${version}"
+    testImplementation 'org.assertj:assertj-core:3.20.2'
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.7.2'
```

Comment:
The above versions seem to be common for both plugins, could we define them in the parent project?

Reply:
oh right, fixed, with other common dependencies.